interface/src/avatar/MyAvatar.cpp

```diff
@@ -1852,6 +1861,21 @@ void MyAvatar::updateOrientation(float deltaTime) {
     }
 
     if (qApp->isHMDMode() && _hmdRollControlEnabled) {
+#ifdef USE_SIMPLE_ROLL_CONTROL
+        float speed = glm::length(getVelocity());
+        const float MIN_CONTROL_SPEED = 0.01f;
+        if (speed >= MIN_CONTROL_SPEED) {
+            // Use head/HMD roll to turn while walking or flying.
+            glm::vec3 avatarUp = getOrientation() * IDENTITY_UP;
+            glm::vec3 rolledRight = getHeadOrientation() * IDENTITY_RIGHT;
+            float rolledRightDotUp = glm::dot(rolledRight, avatarUp);
+            const float MIN_ROLL_DOT_THRESHOLD = 0.15f; // 0.15 corresponds to ~8.62 degrees
+            if (fabsf(rolledRightDotUp) > MIN_ROLL_DOT_THRESHOLD) {
+                rolledRightDotUp -= (rolledRightDotUp > 0.0f) ? MIN_ROLL_DOT_THRESHOLD : - MIN_ROLL_DOT_THRESHOLD;
+                totalBodyYaw += rolledRightDotUp * deltaTime * _hmdRollControlSpeed;
+            }
+        }
```

Comment:
What am I doing here?  I'm computing the dot produce between the head's local "righ axis" and the avatar's "up axis".  No matter the head's current yaw or pitch... the discrepancy between these two vectors can be used as a "yaw throttle" used for roll-control.

Note: there is no need to actually compute the angle since the dot product increases/decreases with it.  That is, we're using `sin(angle)` rather than `angle`.  The relationship isn't strictly linear, but it is for small angles and doesn't deviate too much until we turn our head nearly sideways, which most people won't be doing.